src/ennemy/update_ennemy_move.c

```diff
@@ -0,0 +1,98 @@
+/*
+** EPITECH PROJECT, 2022
+** rpg
+** File description:
+** update ennemy move
+*/
+
+#include <stdlib.h>
+#include "ennemies.h"
+#include "ennemy_pathfind.h"
+
+static int get_data(any_t **rect_speed, any_t **move_speed, any_t *data,
+    any_t **rect_list)
+{
+    if (data == NULL) {
```

Comment:
check if rect_speed is NULL or move_speed too
because *NULL -> seg

Reply:
there are NULL fill them here